src/single-spa-css.ts

```diff
@@ -133,16 +155,11 @@ export default function singleSpaCss<ExtraProps>(
 
 type SingleSpaCssOpts = {
   cssUrls: CssUrl[];
+  webpackExtractedCss?: boolean;
```

Comment:
I tried following the logic for this option but couldn't quite tease out why its needed. Is this so that a consumer can have `webpackExtractedCss: isProduction` in their config, and not have this plugin actually extract the CSS files in dev mode?

Reply:
Yeah it's so that the user of single-spa-css can turn on/off the webpack css assets functionality. There are situations where you'd want to just pass in `cssUrls` without using the automatic webpack css assets - single-spa-css works for non-webpack projects as well.

I've thought about this, though, and wonder if it would make sense to have the default be `true` but then have it be a no-op if the webpack plugin is not installed. However, that might be confusing for situations where you need the automatic webpack assets to be loaded and you set webpackExtractedCss to true but then it does a no-op because you forgot to include the webpack plugin.

Reply:
Oh I see! I thought this was tied to Webpack. Very cool that it'll support other bundlers.